README.md

```diff
@@ -70,3 +70,17 @@ stats.increment('home.page.hits')
 Thread Safety
 -------------
 `DogStatsD` and `ThreadStats` are thread-safe.
+
+Origin detection over UDP
+-------------
+Origin detection is a method to detect which pod DogStatsD packets are coming from in order to add the pod's tags to the tag list.
+
+To enable origin detection over UDP, add the following lines to your application manifest
+```
+env:
+  - name: DD_ENTITY_ID
```

Comment:
There is no other mention to the `DD_ENTITY_ID` environment variable in the docs, and that would be desirable to provide some more context to the users. For instance, in the datadog-go client we have this:
```
- If the `DD_ENTITY_ID` environment variable is found, its value will be injected as a global `dd.internal.entity_id` tag. This tag will be used by the Datadog Agent to insert container tags to the metrics. You should only `append` to the `c.Tags` slice to avoid overwriting this global tag.
```

Reply:
thanks!
this line ```The DogStatsD client attaches an internal tag, `entity_id`. The value of this tag is the content of the `DD_ENTITY_ID` environment variable, which is the pod’s UID.``` just below the example mention the `DD_ENTITY_ID`, I guess I can put it just before the example with context more like the `datadog-go` client description

Reply:
can I get a second review ?